framework/packages/abstract-client/tests/integration.rs

```diff
@@ -1342,3 +1343,28 @@ fn install_same_app_on_different_accounts() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+#[test]
+// TODO: App, Adapter, What are we installing here xd?
+fn install_ibc_client_on_creation() -> anyhow::Result<()> {
+    let chain = MockBech32::new("mock");
+    let client = AbstractClient::builder(chain).build()?;
+
+    let account1 = client
+        .account_builder()
+        // TODO: App, Adapter, What are we installing here xd?
+        .install_app::<IbcClient<MockBech32>>(&cosmwasm_std::Empty {})?
+        .build()?;
+
+    let ibc_module_addr = account1.module_addresses(vec![IBC_CLIENT.to_owned()])?;
+    assert_eq!(ibc_module_addr.modules[0].0, IBC_CLIENT);
+
+    let account2 = client
+        .account_builder()
+        .install_adapter::<IbcClient<MockBech32>>()?
+        .build()?;
+
+    let ibc_module_addr = account2.module_addresses(vec![IBC_CLIENT.to_owned()])?;
+    assert_eq!(ibc_module_addr.modules[0].0, IBC_CLIENT);
+    Ok(())
+}
```

Comment:
added small test that installs ibc_client. Should we consider it just as adapter for users?

Reply:
Very nice ! But you can just use update_setting to do that !

Reply:
Ok, yes, That's an adapter then, no init msg :)